src/results.jl

```diff
@@ -60,26 +58,27 @@ mutable struct Result
             column_type_map[column_number(jl_result, k)] = v
         end
 
-        jl_result.column_types = col_types = collect(Type, imap(enumerate(col_oids)) do itr
-            col_num, col_oid = itr
-            get(column_type_map, col_num) do
-                get(type_lookup, col_oid, String)
-            end
-        end)
+        jl_result.column_types =
+            col_types = collect(Type, imap(enumerate(col_oids)) do itr
+                col_num, col_oid = itr
+                get(column_type_map, col_num) do
+                    get(type_lookup, col_oid, String)
+                end
+            end)
```

Comment:
There's a lot going on here and I feel like something is wrong but I'm not sure exactly what